hadoop-tools/hadoop-aws/src/site/markdown/tools/hadoop-aws/index.md

```diff
@@ -1107,7 +1107,9 @@ options are covered in [Testing](./testing.md).
   <name>fs.s3a.prefetch.block.size</name>
   <value>8MB</value>
   <description>
-      The size of a single prefetched block of data.
+      The size of a single prefetched block of data. 
+      Default value is 8 MB.
+      Lower limit for the block size is 1 byte.
```

Comment:
Sorry, I should have updated my comment. We discussed offline, you can either revert this change completely, or change to just the following:

```suggestion
Decreasing this will increase the number of prefetches required, and may negatively impact performance.  
```